check_license_source_files.sh

```diff
@@ -0,0 +1,202 @@
+#!/bin/bash
+
+# This regular expression can be set in the '.gitlab-ci.yml' file, and is passed
```

Comment:
This file doesn't have `set -e`. It won't actually fail when there is a license problem.